README.md

```diff
@@ -27,6 +27,7 @@ A library for Opt-in and Telemetry data to be sent to the StellarWP Telemetry se
 		- [stellarwp/telemetry/send\_data\_url](#stellarwptelemetrysend_data_url)
 		- [stellarwp/telemetry/last\_send\_expire\_seconds](#stellarwptelemetrylast_send_expire_seconds)
 		- [stellarwp/telemetry/exit\_interview\_args](#stellarwptelemetryexit_interview_args)
+	- [Site Health](#adding-plugin-data-to-site-health)
```

Comment:
🟠 Indentation here

Reply:
I looked at this, not as a Filter Reference but as its own Site Health section as an h2.  Do you think it should be under the Filter Reference section? @tarecord

Reply:
Oh, dang haha. This is fine 😅